webvtt.go

```diff
@@ -32,7 +32,7 @@ var (
 	bytesWebVTTItalicEndTag            = []byte("</i>")
 	bytesWebVTTItalicStartTag          = []byte("<i>")
 	bytesWebVTTTimeBoundariesSeparator = []byte(webvttTimeBoundariesSeparator)
-	webVTTRegexpStartTag               = regexp.MustCompile(`(<v([\.\w]*)([\s\w]+)+>)`)
+	webVTTRegexpStartTag               = regexp.MustCompile(`(<v([\.\w]*)(.{1,}?)>)`)
```

Comment:
Could you replace `{1,}` with `+` ?

Reply:
Thank you for the suggestion. Pushed a new commit.